src/Makefile

```diff
@@ -35,10 +35,12 @@ BINDIR = $(PREFIX)/bin
 ### Built-in benchmark for pgo-builds
 PGOBENCH = ./$(EXE) bench
 
-### Object files
-OBJS = benchmark.o bitbase.o bitboard.o endgame.o evaluate.o main.o \
-	material.o misc.o movegen.o movepick.o pawns.o position.o psqt.o \
-	search.o thread.o timeman.o tt.o uci.o ucioption.o tune.o syzygy/tbprobe.o
+### Source files
+SRCS = benchmark.cpp bitbase.cpp bitboard.cpp endgame.cpp evaluate.cpp main.cpp \
+	material.cpp misc.cpp movegen.cpp movepick.cpp pawns.cpp position.cpp psqt.cpp \
+	search.cpp thread.cpp timeman.cpp tt.cpp uci.cpp ucioption.cpp tune.cpp syzygy/tbprobe.cpp
```

Comment:
Since we're specifying all .cpp files, perhaps wildcards would be more elegant and future proof? Also, defining ``OBJS`` might make more clear what's going on?

```Makefile
SRCS = $(wildcard *.cpp) $(wildcard syzygy/*.cpp)
OBJS = $(notdir $(SRCS:.cpp=.o))
```

Reply:
I guess that's a style question, is there any chance we might create a .cpp file that we don't want in this list in the future? I'm happy either way, I'll leave it up to @vondele which he prefers.

Perhaps I shouldn't have "resolved" the other conversation, I like the OBJS idea, I should have figured that out myself - I'll do that ...

Reply:
I prefer the explicit list of .cpp files. I tend to have random 'test.cpp' around in source trees, or in case of missing files, there will be a rather clear warning.